src/common/timelineEvent.ts

```diff
@@ -39,8 +39,16 @@ export interface CommentEvent {
 	createdAt: string;
 }
 
+export interface reviewResolveInfo {
```

Comment:
The style in this codebase is that interface names are Pascal case.

Reply:
And the signature [here](https://github.com/microsoft/vscode-pull-request-github/blob/9859e75b24f99d8402fc443a7bd286e6ed0bc1c5/webviews/common/context.tsx#L179) should use the new type.